ddl/db_change_test.go

```diff
@@ -114,9 +118,9 @@ func (s *testStateChangeSuite) TestShowCreateTable(c *C) {
 			var result *testkit.Result
 			tbl2 := testGetTableByName(c, tk.Se, "test", "t2")
```

Comment:
Do we need to replace `tk.Se`with `tkInternal.Se`? Otherwise, it also has DATA RACE.

Reply:
Nice idea, `testGetTableByName` won't exec SQL, but get the domain and check the schema.
Both `tk` and `tkInternal` is ok, maybe tkInternal is more elegant, Addressed.